cuML/test/kmeans_test.cu

```diff
@@ -103,8 +103,15 @@ const std::vector<KmeansInputs<float> > inputsf2 = {
 const std::vector<KmeansInputs<double> > inputsd2 = {
 		{ 2, 0.05, 4, 2 }};
 
+
+// FIXME: These tests are disabled due to being too sensitive to RNG:
+// https://github.com/rapidsai/cuml/issues/71
+// From dantegd:
```

Comment:
No need for the quote, the issue is enough

Reply:
done